code/modules/pits/pits.dm

```diff
@@ -0,0 +1,215 @@
+//functions for digging pits in the ground and storing stuff in it
+#define NUMCONTENT 5
+#define NUMBURYTIMES 3
+
+/obj/dugpit
+	name = "pit"
+	anchored = 1
+	icon = 'icons/misc/Pit.dmi'
+	icon_state = "pit"
+	mouse_opacity = 0
+	layer = HIGH_TURF_LAYER
+	var/turf/open/floor/plating/f13/outside/desert/parent
+
+obj/dugpit/New(lnk)
+	..()
+	parent = lnk
+
+/obj/dugpit/proc/dugresist(usr)
```

Comment:
you might be able to remove line 20 by making this instead of `(usr)` make it `(mob/living/user)`